rpc_jobs/rpc_aio.yml

```diff
@@ -122,10 +122,6 @@
          IMAGE: "{IMAGE}"
          FLAVOR: "{FLAVOR}"
          REGION: "{REGION}"
-      - tempest_params:
-         TEMPEST_TEST_SETS: "scenario defcore"
-         RUN_TEMPEST_OPTS: "--serial"
-         TESTR_OPTS: ""
```

Comment:
You'll need to set these in https://github.com/rcbops/rpc-gating/blob/master/playbooks/vars/aio.yml -- specifically, rpc-o sets group_vars/all/rpc-o.yml `tempest_test_sets: "scenario defcore cinder_backup"` but we don't want to run the cinder_backup tests as that's not in our `TEMPEST_TEST_SETS`.